### PR TITLE
chore(explore): Create new entrypoints for Echarts Timeseries

### PR DESCRIPTION
**Pull request by kgabryje**
### SUMMARY
This PR adds the new charts/entry points implemented in https://github.com/apache-superset/superset-ui/pull/1249. For more details, see description.
https://github.com/apache-superset/superset-ui/pull/1249 must be merged before this PR! After that, the superset-ui versions need to be bumped

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
See https://github.com/apache-superset/superset-ui/pull/1249

### TESTING INSTRUCTIONS
1. Create Echarts Area Chart. Verify that you can change series type to Line, Smooth Line, Step Start/Middle/End.
2. Create Line, Smooth Line, Bar, Scatter, Step Start/Middle/End Timeseries chart. Verify that you CANNOT change the series type or select "area".
3. Create Timeseries Chart. Verify that all series types and "area" checkbox are available

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

@junlincc @villebro

### CODE REVIEW

File: superset-frontend/src/visualizations/presets/MainPreset.js [hunk 1]
```diff
@@ -134,9 +141,33 @@ export default class MainPreset extends Preset {
...
+        }),
+        new EchartsTimeseriesStepStartChartPlugin().configure({
+          key: 'echarts_timeseries_step_start',
+        }),
+        new EchartsTimeseriesStepMiddleChartPlugin().configure({
+          key: 'echarts_timeseries_step_middle',
+        }),
+        new EchartsTimeseriesStepEndChartPlugin().configure({
+          key: 'echarts_timeseries_step_end',
+        }),
```
Comment (villebro) [inline superset-frontend/src/visualizations/presets/MainPreset.js:170]: I wonder if these should just collectively be `EchartsTimeseriesStepChartPlugin` with an additional control to define start/middle/end

Comment (kgabryje) [reply superset-frontend/src/visualizations/presets/MainPreset.js:170]: Good question. @junlincc WDYT?

**Comment by villebro** [comment]
/testenv up FEATURE_DASHBOARD_NATIVE_FILTERS=true FEATURE_DASHBOARD_CROSS_FILTERS=true

**Review by geido — APPROVED** [review]
LGTM!

**Review by villebro — APPROVED** [review]
Tested to the best of my ability, and all looks solid

**Comment by villebro** [comment]
/testenv up FEATURE_DASHBOARD_NATIVE_FILTERS=true FEATURE_DASHBOARD_CROSS_FILTERS=true

**Comment by villebro** [comment]
/testenv up FEATURE_DASHBOARD_NATIVE_FILTERS=true FEATURE_DASHBOARD_CROSS_FILTERS=true

**Review by suddjian — APPROVED** [review]
Tested, LGTM

**Comment by rosemarie-chiu** [comment]
🏷 2021.31

**Pull request merged**